.github/workflows/sh-checker.yml

```diff
@@ -0,0 +1,18 @@
+name: sh-checker
+on:
+  - pull_request
+jobs:
+  sh-checker:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Run the sh-checker
+        uses: luizm/action-sh-checker@v0.1.8
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          SHELLCHECK_OPTS: -e SC1004 # It is posible to exclude some shellcheck warnings.
```

Comment:
Do we need this particular exclusion? I know it's in the example action but is it required?

https://github.com/koalaman/shellcheck/wiki/SC1004

Reply:
Ah lol nope not at all. It should pick up shellcheckrc